react-app/src/components/StepperComponent.tsx

```diff
@@ -193,6 +202,40 @@ const StepperComponent = (props: StepperComponentProps) => {
     }
   };
 
+  const lockScreen = (id: String) => {
+    setLockScreen(true);
+    VSCodeAPI.onMessage((message) => {
+      if (message.data.id === id) {
+        console.log('Recived Message');
+        setLockScreen(false);
+      }
+    });
+  };
+
+  const creatScreenLock = () => {
+    return (
+      <div>
+        <Dialog
+          PaperProps={{
+            style: {
+              backgroundColor: 'transparent',
+              boxShadow: 'none',
+              overflow: 'visible',
+            },
+          }}
+          open={true}
+          maxWidth={false}
+        >
+          <ClipLoader
```

Comment:
why not use https://material-ui.com/components/progress/ ?

Reply:
I just wasn’t aware of the component from Material-Ui. I will replace the current Loader. Thank you.